pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -208,12 +209,13 @@ type PostgresStatus struct {
 // makes sense to expose. E.g. pool size (min/max boundaries), max client
 // connections etc.
 type ConnectionPooler struct {
-	NumberOfInstances *int32 `json:"numberOfInstances,omitempty"`
-	Schema            string `json:"schema,omitempty"`
-	User              string `json:"user,omitempty"`
-	Mode              string `json:"mode,omitempty"`
-	DockerImage       string `json:"dockerImage,omitempty"`
-	MaxDBConnections  *int32 `json:"maxDBConnections,omitempty"`
+	NumberOfInstances *int32          `json:"numberOfInstances,omitempty"`
+	Schema            string          `json:"schema,omitempty"`
+	User              string          `json:"user,omitempty"`
+	Mode              string          `json:"mode,omitempty"`
+	DockerImage       string          `json:"dockerImage,omitempty"`
+	MaxDBConnections  *int32          `json:"maxDBConnections,omitempty"`
+	NodeAffinity      v1.NodeAffinity `json:"nodeAffinity,omitempty"`
```

Comment:
Do we need a `NodeAffinity` setting for the pooler, too? @erthalion @Jan-M 
Can we not reuse the affinity defined in the `Postgresql` spec?
If I see it correctly this field isn't even reflected on creation and syncing, e.g. it is not set in `generateConnectionPoolerPodTemplate`.

Reply:
Can you remove the field `NodeAffintity` from pooler API?

Reply:
Will do!